docs/en/index.rst

```diff
@@ -59,6 +59,14 @@ The following configuration should be present in the config array of your **conf
 
             // Whether to store failed jobs in the queue_failed_jobs table. default: false
             'storeFailedJobs' => true,
+
+            // (optional) The cache configuration for storing unique job ids.
+            // `duration` should be greater than the maximum length of time any job can be expected to
+            // remain on the queue. Otherwise, duplicate jobs may be possible. Defaults to +24 hours.
+            // See https://book.cakephp.org/4/en/core-libraries/caching.html#configuring-cache-engines.
+            'uniqueCache' => [
+                'engine' => 'File',
```

Comment:
Should we suggest a safer default? Using files will not ensure uniqueness across multiple servers.

Reply:
Yes, I think that makes sense. I wasn't really considering this example a suggestion, I just went with the simplest example. Maybe this should be called out in the comments above?

Reply:
I think we can mention the File engine as only suitable for local development.

Reply:
👍 I added a note